test/src/code_field/tabs_to_spaces_test.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_code_editor/src/code_field/text_editing_value.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+const _codeRowWithoutTabs = 'int a = 1;';
+const _codeWithTabs = '''
+class Foo {
+\tint field;
+\tvoid method() {
+\t\tprint(field);
+\t}
+}
+''';
+const _codeWithDoubleSpaces = '''
+class Foo {
+  int field;
+  void method() {
+    print(field);
+  }
+}
+''';
+const _spaceCount = 2;
+
+
+void main() {
+  group('Replace tabs with spaces', () {
```

Comment:
1. You can further group 'Empty', 'Without tabs', and 'With tabs'
2. If combined test names do not read as a sentence, add periods after group names. https://medium.com/flutter-senior/how-to-name-and-group-tests-in-a-dart-file-553e194bd8af